src/Host.cpp

```diff
@@ -36,10 +36,11 @@
 #include "mudlet.h"
 
 #include "pre_guard.h"
-#include <QtUiTools>
 #include <QApplication>
 #include <QDir>
 #include <QMessageBox>
+#include <QStringBuilder>
+#include <QtUiTools>
```

Comment:
Thanks for catching this - I used alphabetical sorting on this, not clang-format. Fixed.